src/test/ui/test-attrs/inaccessible-test-modules.stderr

```diff
@@ -11,10 +11,16 @@ error[E0432]: unresolved import `test`
   --> $DIR/inaccessible-test-modules.rs:6:5
    |
 LL | use test as y;
-   |     ----^^^^^
-   |     |
-   |     no `test` in the root
-   |     help: a similar name exists in the module: `test`
+   |     ^^^^^^^^^ no `test` in the root
+   |
+help: a similar name exists in the module
+   |
+LL | use test as y;
```

Comment:
What's this suggesting? Turning `test` into `test`?

Reply:
I think this is the original problem of the test module and not a problem caused by this PR